app/scripts/templates/settings/devices.mustache

```diff
@@ -20,11 +20,14 @@
         {{#devices}}
           <li class="device {{type}}" id="{{id}}">
             <div class="device-name">
-              {{name}} {{#isCurrentDevice}}<span class="device-current">{{#t}}(current){{/t}}</span>{{/isCurrentDevice}}
+              {{name}}
             </div>
+            {{#isCurrentDevice}}<span class="device-current">{{#t}}current device{{/t}}</span>{{/isCurrentDevice}}
```

Comment:
before it was on the same line and on the new line. I like the fixed approach, but if you make `{{#isCurrentDevice}}` be on new lines properly that would also work.